### PR TITLE
new package: py-tempora

### PR DESCRIPTION
**Pull request by marcmengel**
python time utilities, dependency for cherrypy web framework.

### CODE REVIEW

**Comment by marcmengel** [comment]
Depends on #13581

File: var/spack/repos/builtin/packages/py-tempora/package.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from spack import *
+
+
+class PyTempora(PythonPackage):
+    """Objects and routines pertaining to date and time (tempora) """
+
+    homepage = "https://github.com/jaraco/tempora"
+    url      = "https://files.pythonhosted.org/packages/2f/b5/5b0464385454c5ca93a39a1c6acefdf574aeb10ef45fa8958b3832cc7d96/tempora-1.14.1.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tempora/package.py]:
```suggestion
    url      = "https://pypi.io/packages/source/t/tempora/tempora-1.14.1.tar.gz"
```

File: var/spack/repos/builtin/packages/py-tempora/package.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+    version('1.14.1', sha256='cb60b1d2b1664104e307f8e5269d7f4acdb077c82e35cd57246ae14a3427d2d6')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-jaraco-functools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tempora/package.py]: Missing dependencies:
```
python_requires = >=2.7
install_requires =
        six
        pytz
        jaraco.functools>=1.20
setup_requires = setuptools_scm >= 1.15.0
```

File: var/spack/repos/builtin/packages/py-tempora/package.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+class PyTempora(PythonPackage):
+    """Objects and routines pertaining to date and time (tempora) """
+
+    homepage = "https://github.com/jaraco/tempora"
+    url      = "https://pypi.io/packages/source/t/tempora/tempora-1.14.1.tar.gz"
+
+    version('1.14.1', sha256='cb60b1d2b1664104e307f8e5269d7f4acdb077c82e35cd57246ae14a3427d2d6')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tempora/package.py]:
```suggestion
    depends_on('py-setuptools-scm@1.15.0:', type='build')
```

File: var/spack/repos/builtin/packages/py-tempora/package.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    depends_on('py-six', type=('build', 'run'))
+    depends_on('py-pytz', type=('build', 'run'))
+    depends_on('py-jaraco-functools@1.20', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tempora/package.py]:
```suggestion
    depends_on('py-jaraco-functools@1.20:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-tempora/package.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    depends_on('python@2.7', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tempora/package.py]:
```suggestion
    depends_on('python@2.7:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-tempora/package.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+from spack import *
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tempora/package.py]: Missing a license

**Pull request merged**